app/views/user_registrations/link_github.html.slim

```diff
@@ -0,0 +1,19 @@
+.page-header
+  h3.text-center.green-text Welcome on board! Let's get started
+  
+  p.text-center During the program, you will need a Github repository and account to share your awesome code with your mentor.
+
+  p.text-center Don't have a GitHub account and repo? Check this guide on how to create one! 
+
```

Comment:
Also did u made sure that user cant move futher without adding a git link ? 
I mean It is mandatory for user to have a git link. So when they leave empty. Can U display message like "It is mandatory to make a github account before proceeding" @alma-castant Is it fine or you want to change the error message ?